book/src/csi-driver-object.md

```diff
@@ -52,8 +52,13 @@ To install, a CSI driver's deployment manifest must contain a `CSIDriver`
 object as shown in the example above.
 
 >NOTE: The cluster-driver-registrar side-car which was used to create CSIDriver
->objects in Kubernetes 1.13 is being redesigned for Kubernetes 1.15 and
->therefore not available for Kubernetes 1.14.
+>objects in Kubernetes 1.13 has been deprecated for Kubernetes 1.16. No
+>cluster-driver-registrar has been released since Kubernetes 1.14.
```

Comment:
Replace "since" with "for Kubernetes 1.14 and later" to make it clear exactly which versions don't work

Reply:
Replaced with links

Reply:
sorry, just replaced with your suggestion.